py/visdom/server.py

```diff
@@ -1743,6 +1742,32 @@ def download_scripts(proxies=None, install_dir=None):
                 logging.error('Error {} while downloading {}'.format(
                     exc.reason, key))
 
+    # Download MathJax Js Files
+    import requests
+    cdnjs_url = 'https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/' 
+    mathjax_dir='\\'.join(cdnjs_url.split('/')[-3:])
```

Comment:
I'm not sure this is the most effective syntax for what you're trying to do here - to get the os specific path, you can use `os.path.join(*cdnjs_url.split('/')[-3:])`. In general you should avoid assuming details about the os and pathing (which is what I _think_ you're doing with `'\\'`).

Reply:
I did not understood this comment.
The line: `mathjax_dir='\\'.join(cdnjs_url.split('/')[-3:])` will return the value as: mathjax\\2.7.5. 
But if that is not good syntax then I will replace it with `os.path.join()`

Reply:
Changes Done!.....:)

Reply:
`mathjax\2.7.5` is the directory structure only for windows machines. Unix machines use `/` as the directory separator, so we use `os.path.join` rather than directly joining with a character, as the `os.path` library determines whether to use `\` or `/` based on the platform running the script.